nubis/bin/example-ldap2consul.conf

```diff
@@ -0,0 +1,13 @@
+[ldap]
+server = ldap.domain.com
+binduser = somebinduser
+bindpass = agoodpass
+searchbase = dc=domain
+searchfilter = (cn=group)
```

Comment:
searchfilter is very LDAP specific, I would add a simpler groups argument as well:
 groups = [ aws_users, moc_users ]